docs/deployment/cloud-scanning.md

```diff
@@ -0,0 +1,190 @@
+
+After enabling cloud scanning, a full scan starts **immediately** on all the repositories you have added.
+
+Once a repository has been added to Semgrep AppSec Platform, it becomes a **project**. A project in Semgrep AppSec Platform includes all the findings and scan metadata of that repository.
+
+### If the page doesn't display any repositories
+
+1. Ensure you have provided access to **both** the private and public Semgrep GitHub to the repositories you want to scan by following the steps in [Permissions and synchronicity](#permissions-and-synchronicity).
+1. In Semgrep AppSec Platform, click **<i class="fa-solid fa-folder-open"></i> Projects**.
+1. If the page doesn't display the repository you want to add, click **Can't find your project? > Sync projects**.
+1. If the page doesn't display any repositories, click **Sync projects**.
+1. Optional: Perform a hard refresh (<kbd>Ctrl</kbd>+<kbd>F5</kbd> or <kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>R</kbd>).
+
+### How cloud scanning detects repositories
+
+- Repositories with **existing** Semgrep CI jobs are **excluded** from the list. See the following section to convert your existing Semgrep CI jobs to cloud scanning.
+- Repositories must be accessible to both the public Semgrep GitHub app and the private Semgrep GitHub app.
+
+### Convert or migrate an existing Semgrep CI job
+
+If you'd like to migrate a large number of Semgrep CI jobs to cloud scanning, you can reach out to your technical account manager (TAM) for assistance.
+
+<!-- the process requires deleting the project and is not entirely ideal and i'm not sure if we should just say "talk to a tam" -->
+
+1. Delete the project (link-tk)
+1. After deleting the project, the cloud scanning repository list should display the repository you want to convert. Follow the steps in [Add a repository](#add-a-repository).
+
+:::warning
+- Deleting a project also deletes all of its associated history, findings and scan metadata. **This includes triaged findings**. When you re-add the repository, it is treated as a new project.
+:::
+
+<!-- For SME reviewer - is the "treated as a new project" intentional? I tested this functionality out and that's what happened in my case. Answer: yes -->
+
+## Default configuration
+
+By default, projects on cloud scanning are configured with:
+
+- **Weekly full scans** of the entire repository. When a project is first added to cloud scanning, the AppSec Platform performs an initial scan and then sets a random time up to 6 days after to perform a weekly full scan. Each weekly scan occurs on that same day and time.
+- **Diff-aware scans** on pull requests that run on every PR. These diff-aware scans follow the **rule modes** set in your Policies, ensuring that developers are only notified of findings from high-signal rules you place in Comment or Block mode.
+
+## Scan management and configuration
+
+<!-- I haven't been able to see ANY of these in my ZCS deployment so there are no screenshots. -->
```

Comment:
None of the buttons (on demand scan through **Run a new scan**, for example) are appearing for me. FYI.